project-base/templates/Front/Content/Product/detail.html.twig

```diff
@@ -6,6 +6,8 @@
 {% set descriptionHeadline = 'Product information'|trans %}
 {% set parametersHeadline = 'Technical parameters'|trans %}
 {% set accessoriesHeadline = 'Accessories'|trans %}
+{% set variants = productView.variants %}
+{% set accessories = productView.accessories %}
```

Comment:
_nitpick_ 🤓 

I'm thinking about removing variable and use `productView.xxx` directly. 
To me, it's more obvious, I will leave the decision to you

Reply:
I agree but not sure if removing them cannot cause BC break in case you have extended templates.

Reply:
It's project-base template. I don't think it will be extended

Reply:
You are right! It is gone 🎉